composer.json

```diff
@@ -45,8 +52,7 @@
         "symfony/expression-language":    "~2.7|^3.0",
         "symfony/css-selector":           "^2.7|^3.0",
         "phpoption/phpoption":            "^1.1",
-        "jms/serializer-bundle":          "^1.0",
-        "sllh/php-cs-fixer-styleci-bridge": "^1.3"
+        "jms/serializer-bundle":          "dev-symfony_3_0 as 1.0"
```

Comment:
needs to be reverted back once JMSSerializerBundle gets a new release after https://github.com/schmittjoh/JMSSerializerBundle/pull/492 was merged